[2024-10-02T20:22:31.107+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T20:22:31.122+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_forest_classifier_dag.rfc_save_results manual__2024-10-02T19:56:16.733472+00:00 [queued]>
[2024-10-02T20:22:31.129+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_forest_classifier_dag.rfc_save_results manual__2024-10-02T19:56:16.733472+00:00 [queued]>
[2024-10-02T20:22:31.130+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T20:22:31.141+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): rfc_save_results> on 2024-10-02 19:56:16.733472+00:00
[2024-10-02T20:22:31.151+0000] {standard_task_runner.py:64} INFO - Started process 13246 to run task
[2024-10-02T20:22:31.154+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'random_forest_classifier_dag', 'rfc_save_results', 'manual__2024-10-02T19:56:16.733472+00:00', '--job-id', '710', '--raw', '--subdir', 'DAGS_FOLDER/***/dags/.ipynb_checkpoints/hw1-checkpoint.py', '--cfg-path', '/tmp/tmp20gbt9vr']
[2024-10-02T20:22:31.156+0000] {standard_task_runner.py:91} INFO - Job 710: Subtask rfc_save_results
[2024-10-02T20:22:31.201+0000] {task_command.py:426} INFO - Running <TaskInstance: random_forest_classifier_dag.rfc_save_results manual__2024-10-02T19:56:16.733472+00:00 [running]> on host 4b5f1410d111
[2024-10-02T20:22:31.295+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexvlg34@gmail.com' AIRFLOW_CTX_DAG_OWNER='Alex Trigolos' AIRFLOW_CTX_DAG_ID='random_forest_classifier_dag' AIRFLOW_CTX_TASK_ID='rfc_save_results' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T19:56:16.733472+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-02T19:56:16.733472+00:00'
[2024-10-02T20:22:31.296+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T20:22:31.310+0000] {hw1-checkpoint.py:167} INFO - Save metrics...
[2024-10-02T20:22:31.398+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2024-10-02T20:22:31.399+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-10-02T20:22:31.965+0000] {hw1-checkpoint.py:194} INFO - DONE!!!
[2024-10-02T20:22:31.967+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T20:22:31.967+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T20:22:31.976+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_forest_classifier_dag, task_id=rfc_save_results, run_id=manual__2024-10-02T19:56:16.733472+00:00, execution_date=20241002T195616, start_date=20241002T202231, end_date=20241002T202231
[2024-10-02T20:22:32.050+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-10-02T20:22:32.075+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T20:22:32.078+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
